lib/manager/maven/extract.ts

```diff
@@ -295,10 +300,11 @@ export function resolveParents(packages: PackageFile[]): PackageFile[] {
 }
 
 function cleanResult(
-  packageFiles: PackageFile<Record<string, any>>[]
+  packageFiles: MavenPackageFile[]
 ): PackageFile<Record<string, any>>[] {
   packageFiles.forEach((packageFile) => {
     delete packageFile.mavenProps; // eslint-disable-line no-param-reassign
+    delete packageFile.parent; // eslint-disable-line no-param-reassign
```

Comment:
Isn't this used by datasource?

Reply:
If so then types didn't catch it. Will check

Reply:
Not used